### PR TITLE
[FLOC-4462] Release Flocker 1.14.0

### PR DESCRIPTION
**Pull request by wallrj**
Fixes: https://clusterhq.atlassian.net/browse/FLOC-4462

### CODE REVIEW

**Comment by wallnerryan** [comment]
@wallrj noticed FLOC-338, did this make it in or is 16.04 support not in this release?

**Comment by wallrj** [comment]
> @wallrj noticed FLOC-338, did this make it in or is 16.04 support not in this release?

Yep, it's in there: https://clusterhq.atlassian.net/browse/FLOC-4338 but I'd already added the release note. See the first bullet.

**Comment by wallrj** [comment]
@wallnerryan: The acceptance test builds have all finished now...and the only one still failing is Acceptance tests on Rackspace Ubuntu 14.04:
 * http://ci-live.clusterhq.com:8080/job/ClusterHQ-flocker/job/release-release-flocker-1.14.0/job/_run_acceptance_on_Rackspace_Ubuntu_Trusty_with_Cinder/4/

And I think that's because of FLOC-2966 (floating IPv6 addresses) or something similar.

* FLOC-2966*- [FLOC-2966] Floating IPs prevent attaching volumes to OpenStack nodes - Flocker

The Rackspace instances all seem to have a public IPv6 address assigned.
(perhaps they always did)

But now, in Ubuntu 14.04, that IPv6 address doesn't appear to be configured.

Ubuntu 14.04 does seem to have IPv6 turned on. e.g

```
root@acceptance-test-buildbot-2k3v3vb37undc-1:~# ip addr
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default 
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
    inet 127.0.0.1/8 scope host lo
       valid_lft forever preferred_lft forever
    inet6 ::1/128 scope host 
       valid_lft forever preferred_lft forever
2: eth0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc pfifo_fast state UP group default qlen 1000
    link/ether bc:76:4e:04:62:79 brd ff:ff:ff:ff:ff:ff
    inet 172.99.117.162/24 brd 172.99.117.255 scope global eth0
       valid_lft forever preferred_lft forever
    inet6 fe80::be76:4eff:fe04:6279/64 scope link 
       valid_lft forever preferred_lft forever
3: eth1: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc pfifo_fast state UP group default qlen 1000
    link/ether bc:76:4e:04:62:93 brd ff:ff:ff:ff:ff:ff
    inet 10.210.97.173/19 brd 10.210.127.255 scope global eth1
       valid_lft forever preferred_lft forever
    inet6 fe80::be76:4eff:fe04:6293/64 scope link 
       valid_lft forever preferred_lft forever
4: docker0: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc noqueue state DOWN group default 
    link/ether 02:42:17:e5:88:0b brd ff:ff:ff:ff:ff:ff
    inet 172.17.0.1/16 scope global docker0
       valid_lft forever preferred_lft forever

```

So perhaps the problem is that the IPv6 address is a floating address.

I think we should go ahead with the release anyway.

File: docs/releasenotes/index.rst [hunk 1]
```diff
@@ -9,21 +9,27 @@ You can learn more about where we might be going with future releases by:
 * Stopping by the ``#clusterhq`` channel on ``irc.freenode.net``.
 * Visiting our GitHub repository at https://github.com/ClusterHQ/flocker.
 
-Next Release
+This Release
 ============
 
 v1.14.0
```
Comment (wallnerryan) [inline docs/releasenotes/index.rst]: we need to remember to put in which docker version this release has been tested on. please add which version its been testing on. I think its will only be 1.12 correct?

**Comment by wallnerryan** [comment]
@wallrj release notes look good with one small nit, need to put docker version tested against. Please add this and merge when you are happy. I know we are looking into the build failure but looks like its the ongoing floating ip address with openstack issue so we are not introducing any new bugs.

👍 

R

**Pull request merged**